terraform/modules/data-share-service/sns_alarm.tf

```diff
@@ -0,0 +1,34 @@
+locals {
+  notification_emails = []
+  prefix              = "${var.environment}-gdx-data-share"
+}
+
+resource "aws_cloudformation_stack" "sns_alarm_topic" {
+  name = "${local.prefix}-alarms-topic"
+
+  template_body = jsonencode(
+    {
+      "AWSTemplateFormatVersion" : "2010-09-09",
+      "Resources" : {
+        "EmailSNSTopic" : {
+          "Type" : "AWS::SNS::Topic",
+          "Properties" : {
+            "DisplayName" : "Softwire - ${local.prefix}-alarms",
+            "Subscription" : [
+              for address in local.notification_emails : {
+                Endpoint = address
+                Protocol = "email"
+              }
+            ]
+          }
+        }
+      },
+      "Outputs" : {
+        "ARN" : {
+          "Description" : "Email SNS Topic ARN",
+          "Value" : { "Ref" : "EmailSNSTopic" }
+        }
+      }
+    }
+  )
```

Comment:
Are we using this because it's easier than setting up a topic and topic subscription in tf?

Reply:
This is from me, because of the old situation where you couldn't at all sub an sns topic using terraform via email, however it looks possible now looking at https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sns_topic_subscription

Reply:
Have given this a go!